packages/client/src/links/HTTPBatchLinkOptions.ts

```diff
@@ -6,6 +6,7 @@ import type { HTTPHeaders, Operation } from './types';
 export type HTTPBatchLinkOptions<TRoot extends AnyClientTypes> =
   HTTPLinkBaseOptions<TRoot> & {
     maxURLLength?: number;
+    batchInterval?: number;
```

Comment:
This could be

```ts
HTTPLinkBaseOptions<TRoot> & DataLoaderOptions & { maxURLLength?: '..' }
```
that way, it inherits whatever docs and options we add on dataloader